includes/class-wsuwp-people-directory-page-template.php

```diff
@@ -761,21 +785,29 @@ public function directory_data( $post_id ) {
 		$layout = get_post_meta( $post_id, '_wsu_people_directory_layout', true );
 		$show_photos = get_post_meta( $post_id, '_wsu_people_directory_show_photos', true );
 		$open_profiles_as = get_post_meta( $post_id, '_wsu_people_directory_profile', true );
+
+		// Set up the wrapper classes.
 		$wrapper_classes = 'wsu-people-wrapper';
 		$wrapper_classes .= ( $layout ) ? ' ' . esc_attr( $layout ) : ' table';
-		$theme_template = $this->theme_has_listing_template();
+
+		if ( 'yes' === $show_photos ) {
+			$wrapper_classes .= ' photos';
+		}
+
+		if ( 'lightbox' === $open_profiles_as ) {
+			$wrapper_classes .= ' lightbox';
+		}
+
+		// Set up the page content.
 		$elements = array(
 			'people' => array(),
 			'locations' => array(),
 			'orgs' => array(),
 		);
 		$loading = false;
 
-		// Loop through the local records in their display order and add an `include`
-		// parameter to the request URL for each one, then leverage `orderby=include`
-		// to retrieve the primary records in the desired order. Silly, but effective.
 		if ( $ids ) {
-			$cache_key = md5( get_the_modified_date( 'Y-m-d H:i:s' ) );
+			$cache_key = get_the_modified_date( 'Y-m-d H:i:s' );
```

Comment:
Memcached can treat spaces in cache keys strangely. I think it may just ignore the part of the key after the space, which could lead to some weird results. I'd recommend `md()` there and going with the hash unless it was causing problems.